src/components/organization/IFXOrganizationList.vue

```diff
@@ -32,6 +33,24 @@ export default {
     },
   },
   methods: {
+    getSetItems() {
+      // TODO: make this consistent, no api endpoint should be returning .data
+      return (
+        this.apiRef
+          .getSkinnyList()
+          .then((items) => {
+            if (has(items, 'data')) {
```

Comment:
Rather than use `has`, can you do something like ```
if (Array.isArray(items.data)) {
  this.items = items
} else {
console.error('getList should return a list of formatted objects')
}

Reply:
Good question.  I think this is something Andy did that I copied.  How about items?.data?.length

Reply:
Just gonna kill it.  Not that useful